src/main/java/ticTacToe/grid/Grid.java

```diff
@@ -103,6 +104,6 @@ private int positionFor(String cellNumber) {
     }
 
     private boolean cellDifferentFromMark(String cell) {
-        return !cell.equals(CROSS.mark) && !cell.equals(NOUGHT.mark);
+        return !cell.equals(CROSS.sign) && !cell.equals(NOUGHT.sign);
```

Comment:
Is this trying to test if the current `cell` is  Mark or not?
You could use `Mark.isValidMark(cell)` right?